worlds/mlss/Rules.py

```diff
@@ -0,0 +1,278 @@
+from worlds.generic.Rules import add_rule, forbid_item
+from BaseClasses import MultiWorld
+from .Names.LocationName import LocationName
+from .Locations import all_locations, hidden
+from . import StateLogic
+
+
+def set_rules(world: MultiWorld, player: int, excluded):
```

Comment:
As you can see with #2749 existing, we are trying to discourage and discontinue the use of `world: MultiWorld` in functions. Best options are to either rename to `multiworld` or swap out the Multiworld object for your world object and access it with `world.multiworld`.

Reply:
Changed all references to multiworld in newest commit